tests/utils/synthetic_utils.py

```diff
@@ -0,0 +1,139 @@
+from typing import Any, Dict, Optional
+
+import pytest
+
+from composer.datasets import GLUEHparams, LMDatasetHparams
+from composer.datasets.hparams import DatasetHparams, SyntheticHparamsMixin
+from composer.datasets.synthetic_lm import generate_synthetic_tokenizer
+from composer.models import DeepLabV3Hparams, ModelHparams, TransformerHparams
+
+
+def configure_dataset_for_synthetic(dataset_hparams: DatasetHparams,
+                                    model_hparams: Optional[ModelHparams] = None) -> None:
+    if not isinstance(dataset_hparams, SyntheticHparamsMixin):
+        pytest.xfail(f"{dataset_hparams.__class__.__name__} does not support synthetic data or num_total_batches")
+
+    assert isinstance(dataset_hparams, SyntheticHparamsMixin)
+
+    dataset_hparams.use_synthetic = True
+
+    if isinstance(model_hparams, TransformerHparams):
+        model_hparams_name = type(model_hparams).__name__
+
+        if model_hparams_name not in _model_hparams_to_tokenizer_family:
```

Comment:
See below re: using the class instead of the class name as the dict key

Reply:
Done!